blueoil/cmd/init.py

```diff
@@ -371,26 +362,27 @@ def ask_questions():
 please modify manually after config exported.)"
 
                 all_augmentor[name + default_str] = {"name": name, "defaults": default_val}
-                checkboxes.append({"name": name + default_str, "value": name})
-        data_augmentation_question = {
-            'type': 'checkbox',
-            'name': 'value',
-            'message': 'Please choose augmentors:',
-            'choices': checkboxes
-        }
-        data_augmentation_res = prompt(data_augmentation_question)
-        data_augmentation = {}
+                checkboxes.append(name + default_str)
+        data_augmentation_question = [
+            inquirer.Checkbox(name='value',
+                              message='Please choose augmentors',
+                              choices=checkboxes)
+        ]
+        data_augmentation_res = prompt(data_augmentation_question, None)
         if data_augmentation_res:
             for v in data_augmentation_res:
                 data_augmentation[all_augmentor[v]["name"]] = all_augmentor[v]["defaults"]
 
-    quantize_first_convolution_question = {
-        'type': 'rawlist',
-        'name': 'value',
-        'message': 'apply quantization at the first layer?',
-        'choices': ['yes', 'no']
-    }
-    quantize_first_convolution = prompt(quantize_first_convolution_question)
+    quantize_first_convolution_question = [
+        inquirer.Confirm(name='value',
+                         message='apply quantization at the first layer?',
+                         default=True
+                        )
+    ]
+    if prompt(quantize_first_convolution_question, None):
```

Comment:
How about
`quantize_first_convolution  = prompt(quantize_first_convolution_question, None)`

Reply:
I thought so.  I did this to match the parameter in the old code (yes,no).

I will change the parameter to get True, False instead.

Reply:
I found that the old code is not working properly as I reported in #884 .

Then, I changed to `quantize_first_convolution = prompt(quantize_first_convolution_question, None)` and it works properly now.